### PR TITLE
fix(EU Covid Certificate): [IAGP-57] Change locales for photolibrary permission

### PR DESCRIPTION
**Pull request by fabriziofff**
## Short description
This pr changes the locales for the photo library permission.

Before            |  After
:-------------------------:|:-------------------------:
|  

## List of changes proposed in this pull request
- Update locales
- Added @thisisjp as CODEOWNER for iOS native locales

### CODE REVIEW

File: ios/ItaliaApp/it.lproj/InfoPlist.strings [hunk 1]
```diff
@@ -2,5 +2,5 @@
 "NSCalendarsUsageDescription" = "IO ha necessità di accedere ai tuoi calendari per permetterti di ricordare le scadenze";
 "NSContactsUsageDescription" = "IO ha necessità di accedere ai tuoi contatti per permetterti di aggiungerli come partecipanti alle scadenze";
 "NSFaceIDUsageDescription" = "IO ha necessità di accedere a Face ID per permetterti di effettuare un'autenticazione più veloce";
-"NSPhotoLibraryUsageDescription" = "IO ha necessità di accedere alle foto per la scansione dei codici di pagamento";
+"NSPhotoLibraryUsageDescription" = "per salvare o caricare codici e screenshot";
```
Comment (thisisjp) [inline ios/ItaliaApp/it.lproj/InfoPlist.strings:5]:
```suggestion
"NSPhotoLibraryUsageDescription" = "Con IO puoi salvare bonus e certificati, o caricare screenshot e avvisi di pagamento";
```

File: ios/ItaliaApp/Info.plist [hunk 2]
```diff
@@ -77,7 +77,7 @@
     <key>NSPhotoLibraryAddUsageDescription</key>
     <string>The app needs NSPhotoLibraryAddUsageDescription permission</string>
     <key>NSPhotoLibraryUsageDescription</key>
-    <string>IO needs access to the Photos to scan QR codes</string>
+    <string>to save or load codes and screenshots </string>
```
Comment (thisisjp) [inline ios/ItaliaApp/Info.plist:80]:
```suggestion
    <string>With IO you can save your bonuses and certificates, in addition to uploading screenshots or payment notices</string>
```

File: locales/en/index.yml [hunk 3]
```diff
@@ -1269,7 +1269,7 @@ wallet:
       message: "IO app needs Camera usage permission to shoot at the QR code"
     settingsAlert:
       title: It seems access to Photos has been denied
-      message: IO needs access to the Photos to scan QR codes
+      message: IO needs access your images to save or load codes and screenshots
```
Comment (thisisjp) [inline locales/en/index.yml:1272]:
```suggestion
      message: IO needs access to your gallery to save bonuses and certificates, in addition to uploading screenshots or payment notices
```

File: locales/it/index.yml [hunk 4]
```diff
@@ -1294,7 +1294,7 @@ wallet:
       message: "IO richiede di poter utilizzare la fotocamera ed inquadrare il QR code"
     settingsAlert:
       title: Sembra che l'accesso alle foto sia stato negato
-      message: IO ha bisogno di accedere alle foto per scansionare i codici QR
+      message: IO ha bisogno di accedere alle tue immagini per salvare o caricare codici e screenshot
```
Comment (thisisjp) [inline locales/it/index.yml:1297]:
```suggestion
      message: IO ha bisogno di accedere alla tua galleria per salvare bonus e certificati, o caricare screenshot e avvisi di pagamento
```

**Review by thisisjp — CHANGES_REQUESTED** [review]

**Review by Undermaken — APPROVED** [review]

**Comment by Undermaken** [comment]
@thisisjp merge on your approval

**Review by thisisjp — APPROVED** [review]

**Review by pietro909 — APPROVED** [review]

**Pull request merged**